src/lib/Session.svelte

```diff
@@ -445,7 +467,33 @@
     {#if exitReason !== null}
       <div class="text-red-400">{exitReason}</div>
     {:else if connected}
-      <div class="text-green-400">You are connected!</div>
+      <div class="flex">
+        <div class="text-green-400">You are connected!</div>
+        {#if userId && !hasWriteAccess}
+          <div
+            class="bg-yellow-900 text-yellow-200 px-1 py-0.5 rounded ml-3 inline-flex items-center gap-1"
+          >
+            <svg
+              class="w-3.5 h-3.5"
+              xmlns="http://www.w3.org/2000/svg"
+              width="24"
+              height="24"
+              viewBox="0 0 24 24"
+              fill="none"
+              stroke="currentColor"
+              stroke-width="2"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              ><path d="M1 12s4-8 11-8 11 8 11 8-4 8-11 8-11-8-11-8z" /><circle
+                cx="12"
+                cy="12"
+                r="3"
+              /></svg
+            >
```

Comment:
Sorry this wasn't clear but this icon is in feather icons, you can import it from `svelte-feather-icons` like the other icons in the project!